tests/testthat/test-bias.R

```diff
@@ -108,14 +195,21 @@ test_that("quantile bias and range bias have the same result", {
   quantiles <- c(0.01, 0.025, seq(0.05, 0.95, 0.05), 0.975, 0.99)
   true_value <- 8062
```

Comment:
I think the test doesn't work as expected and the only reason it passes is that the true_value is so far above all predictions that the result is always -1.

Reply:
Suggest to update the test as follows: 

```
test_that("bias_quantile and bias_range() give the same result", {
  predictions <- sort(rnorm(23))
  lower <- rev(predictions[1:12])
  upper <- predictions[12:23]

  range <- c(0, 10, 20, 30, 40, 50, 60, 70, 80, 90, 95, 98)
  quantiles <- c(0.01, 0.025, seq(0.05, 0.95, 0.05), 0.975, 0.99)
  true_value <- rnorm(1)

  range_bias <- bias_range(
    lower = lower, upper = upper,
    range = range, true_value = true_value
  )
  range_quantile <- bias_quantile(
    predictions = predictions, quantiles = quantiles,
    true_value = true_value
  )
  expect_equal(range_bias, range_quantile)
})
```

Reply:
Silly claude (🙈) will update ty

Reply:
Swapped out in https://github.com/epiforecasts/scoringutils/pull/305/commits/06d1fe0ffe7dce8dcacc03a5242af90cfcf2f114